docker/release_dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+
+ENV IROHA_HOME /opt/iroha
+ENV IROHA_BUILD /opt/iroha/build
+
+COPY iroha.deb /tmp/iroha.deb
+RUN apt-get update;\
+    apt-get upgrade; \
+    apt-get install -y \
+        libc-ares-dev \
+        gdebi-core
+
+#Install iroha
+RUN gdebi /tmp/iroha.deb
+
+CMD ["bash"]
```

Comment:
Consider such container: https://github.com/kylemanna/docker-openvpn#quick-start
We can also run `irohad` here and attach required files with volumes before running this container, so there is no need to manually run `irohad`.

Reply:
Now we do not have opportunity to `just run` irohad. After config will be implemented this will be changed.